docs/HACKING.md

```diff
@@ -1,39 +1,37 @@
 # Goldstone Server Hacking Guide
 
 
-This explains how to install and run Goldstone Server locally (mostly in docker containers), so you can do code development on the project.  The instructions assume a Mac OS X Yosemite development environment with [homebrew](http://brew.sh/) and [Docker Toolbox](http://www.docker.com/toolbox) installed.  
+This explains how to install and run Goldstone Server locally, so you can do code development on the project.  These instructions are written for a Mac OS X Yosemite development.
 
-**_Note: if you have manually installed docker-machine and docker-compose, make sure your docker-machine VM name is 'default' in order to be compatible with the supporting scripts._**
+## Prerequisites
 
-[TOC]
+Install [Docker Toolbox](https://github.com/docker/toolbox/releases). 
 
-## Prerequisites
+**_Note: there is an issue with release 1.8.2a of Docker Toolbox/1.4.1 of Docker Compose, but earlier versions work_**
```

Comment:
Since the instructions require installing Docker Toolbox, this Note should refer to only a Docker Toolbox version number.  I'd drop the "/1.4.1 of Docker Compose".